mappers/mqtt-mapper/driver/driver.go

```diff
@@ -0,0 +1,443 @@
+package driver
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/kubeedge/mapper-framework/pkg/common"
+	"gopkg.in/yaml.v3"
```

Comment:
Please sort the import, those begin with `github.com/kubeedge` should be placed in the third part

Reply:
Thank you for your feedback! I have adjusted the order of the imports as you requested.